### PR TITLE
Rename template to blue print

### PR DESCRIPTION
**Pull request by jscharber**
fixes github.com/pavedroad-io/roadctl#120

### CODE REVIEW

File: microservices/ga/datamgr/README.md [hunk 1]
```diff
@@ -200,7 +201,7 @@ Help output:
   Choose a command run in films:
 
   compile         Compile the binary.
-  clean           Remove dep, vendor, binary(s), and execute go clean
+  clean           Remove dep, vendor, binary(s), and executes go clean
```
Comment (jscharber) [inline microservices/ga/datamgr/README.md:203]: @MarkGreenPR did you change this in your branch to modules?

File: microservices/ga/datamgr/README.md [hunk 2]
```diff
@@ -96,6 +96,7 @@ Flags:
   -h, --help               help for roadctl
       --password string    HTTP basic auth password
       --blueprints string  Set the location of the directory holding roadctl blueprints
+>>>>>>> master
```
Comment (jscharber) [inline microservices/ga/datamgr/README.md]: @jscharber  fix this

File: microservices/ga/datamgr/README.md (hunk 1 above)
Comment (MarkGreenPR) [reply microservices/ga/datamgr/README.md:203]: Yes, in Makefile it is now:
Remove binary(s) and execute go clean

**Review by MarkGreenPR — APPROVED** [review]
Do you need the --remove-orphans in the workerPool Makefile?

Otherwise looks like a fairly simple merge.

Maybe merge my branch into master and look at diffs from that?

**Pull request merged**